operations/security_and_authentication/Change_SMNP_credentials_on_Leaf_switches.md

```diff
@@ -0,0 +1,112 @@
+# Change SMNP credentials on Leaf switches
```

Comment:
Capitalization change: "Change SMNP Credentials on Leaf Switches"

Reply:
Should be resolved fixed now.